docs/PUBLISHING.md

```diff
@@ -4,6 +4,42 @@ Publishing Etcher
 This is a small guide to package and publish Etcher to all supported operating
 systems.
 
+Release Types
+-------------
+
+Etcher supports **production** and **snapshot** releases. Each is published to
+a different S3 bucket, and production releases are code signed, while snapshot
+releases aren't and include a short git commit-hash as a build number. For
+example, `1.0.0-beta.19` is a production release, while `1.0.0-beta.19+531ab82`
+is a snapshot release.
+
+In terms of comparison: `1.0.0-beta.19` (production) < `1.0.0-beta.19+531ab82`
+(snapshot) < `1.0.0-rc.1` (production) < `1.0.0-rc.1+7fde24a` (snapshot) <
+`1.0.0` (production) < `1.0.0+2201e5f` (snapshot). Keep in mind that if you're
+running a production release, you'll only be prompted to update to production
+releases, and if you're running a snapshot release, you'll only be able to
+update to other snapshot releases.
+
+The build system creates (and publishes) snapshot releases by default, but you
+can build a specific release type by setting the `RELEASE_TYPE` make variable.
+For example:
+
+```sh
+make <target> RELEASE_TYPE=snapshot
+make <target> RELEASE_TYPE=production
+```
+
+We can control the version range a specific Etcher version will consider when
+showing the update notification dialog by tweaking the `updates.semverRange`
+property of `package.json`.
+
+Update Channels
+---------------
+
+Etcher has a setting to include the unstable update channel. If this option is
+set, Etcher will also consider non-production releases when showing the update
```

Comment:
I'm going to ask you to change the language here, because we need to make a distinction between stable/unstable and snapshot/production, and referring to unstable as "non-production" will confuse people no end!